.github/workflows/release.yml

```diff
@@ -126,8 +126,11 @@ jobs:
             --config=remote-ci-macos \
             --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
             //:ios_xcframework
-      - name: 'Move Envoy.xcframework.zip'
-        run: mv bazel-bin/library/swift/Envoy.xcframework.zip .
+      - name: 'Recompress Envoy.xcframework.zip' # Recompress as bazel uses zip as a container with no compression
```

Comment:
really? seems like a bug we should fix upstream since that zip is always meant for distribution

Reply:
Yes, really:

```
$ du -h bazel-bin/library/swift/Envoy.xcframework.zip
1.0G	bazel-bin/library/swift/Envoy.xcframework.zip
```